lab/lab1/part3-data-transformation.html

```diff
@@ -63,14 +63,52 @@
 
     ===================== */
 
-    var data = bikeArrayClean;
+    var data = bikeArrayDirty;
 
     /* =====================
 
     Start code to filter data
 
     ===================== */
-
+    // Clean Data
```

Comment:
I think this is extremely close. But I think with your comments... you've stopped it from working

I get some errors when I try to load it:


![image](https://user-images.githubusercontent.com/8103418/52544767-ec98a000-2d80-11e9-9d8f-c13b58699db8.png)

Reply:
I uncommented out `BikeArrayClean` but then got another error...

Can you fix this so it works when someone else runs it?

Thanks.